src/toolchain.rs

```diff
@@ -291,15 +304,15 @@ mod tests {
             TARGET_X86_APPLE,
             TARGET_X86_LINUX,
         ] {
-            let input = channel::NIGHTLY.to_owned() + "-" + DATE + "-" + target;
-            assert!(OfficialToolchainDescription::from_str(&input).is_err());
-            // TODO: uncomment once specifying date and target is supporting
-            //   assert_eq!(
-            //       desc.name,
-            //       DistToolchainName::from_str(channel::NIGHTLY).unwrap()
-            //   );
-            //   assert_eq!(desc.date.unwrap().to_string(), DATE);
-            //   assert_eq!(desc.target.unwrap().to_string(), target);
+            let toolchain = channel::NIGHTLY.to_owned() + "-" + DATE + "-" + target;
+            assert!(OfficialToolchainDescription::from_str(&toolchain).is_err());
+            // TODO: Uncomment once target specification is supported
```

Comment:
nit: add the issue link if there is one.

Reply:
Ah, please ignore this comment! Honestly I don't even remember why it was there originally now because it was quite long ago but it was meant to make tests pass prior to some (now) merged changes, which weren't ready before. Targets should be automatically generated for the host machine anyway, so this shouldn't be a blocker for specifying dates.